plugins/main/server/controllers/wazuh-api.ts

```diff
@@ -734,6 +734,8 @@ export class WazuhApiCtrl {
     request: OpenSearchDashboardsRequest,
     response: OpenSearchDashboardsResponseFactory,
   ) {
+    const appConfig = await context.wazuh_core.configuration.get();
+    const reportMaxRows = appConfig['reports.csv.maxRows'] || 10000;
```

Comment:
prefer use constants instead of hard-coded values
```suggestion
  const maxRows = appConfig.data[REPORTS_CSV_MAX_ROWS_KEY] || REPORTS_CSV_MAX_ROWS_DEFAULT;
```